packages/server/tests/MyProfileInformation.test.ts

```diff
@@ -0,0 +1,75 @@
+import Database from '@ioc:Adonis/Lucid/Database';
+import {
+    GetMyProfileInformationRequestBody,
+    GetMyProfileInformationResponseBody,
+} from '@musicroom/types';
+import User from 'App/Models/User';
+import { datatype, internet } from 'faker';
+import test from 'japa';
+import supertest from 'supertest';
+import urlcat from 'urlcat';
+import { USER_ROUTES_GROUP_PREFIX } from '../start/routes';
+import { BASE_URL, initTestUtils } from './utils/TestUtils';
+
+test.group('Users Profile information tests', (group) => {
+    const { createUserAndGetSocket, createSocketConnection } = initTestUtils();
+
+    group.beforeEach(async () => {
+        await Database.beginGlobalTransaction();
+    });
+
+    group.afterEach(async () => {
+        await Database.rollbackGlobalTransaction();
+    });
```

Comment:
If you create sockets in tests, I think you should ensure they are all cleared before running next tests.

```suggestion
    group.beforeEach(async () => {
        initSocketConnection();
        await Database.beginGlobalTransaction();
    });

    group.afterEach(async () => {
        await disconnectEveryRemainingSocketConnection();
        sinon.restore();
        await Database.rollbackGlobalTransaction();
    });
```

Reply:
Yes you're right ! forgot to add required test socket life cycle method here
thanks